Testscripts/Linux/perf_apache.sh

```diff
@@ -0,0 +1,255 @@
+#!/bin/bash
+
+#######################################################################
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+#######################################################################
+
+#######################################################################
+#
+# perf_apache.sh
+# Description:
+#    Install apache and run apache performance tests.
+#    This script needs to be run on client VM.
+#
+# Supported Distros:
+#    Ubuntu/Suse/Centos/RedHat/Debian
+#######################################################################
+
+CONSTANTS_FILE="./constants.sh"
+UTIL_FILE="./utils.sh"
+
+. ${CONSTANTS_FILE} || {
```

Comment:
This is redundant. Constants are exported in utils.sh

Reply:
Fixed